java/ingest/src/main/java/sleeper/ingest/impl/StandardIngestCoordinator.java

```diff
@@ -55,25 +57,28 @@ public static IngestCoordinator<Record> directWriteBackedByArrow(IngestPropertie
                                                                      long maxBatchArrowBufferAllocatorBytes) {
         return builder()
                 .fromProperties(ingestProperties)
-                .backedByArrow()
-                .arrowBufferAllocator(arrowBufferAllocator)
-                .maxNoOfRecordsToWriteToArrowFileAtOnce(maxNoOfRecordsToWriteToArrowFileAtOnce)
-                .workingArrowBufferAllocatorBytes(workingArrowBufferAllocatorBytes)
-                .minBatchArrowBufferAllocatorBytes(minBatchArrowBufferAllocatorBytes)
-                .maxBatchArrowBufferAllocatorBytes(maxBatchArrowBufferAllocatorBytes)
-                .maxNoOfBytesToWriteLocally(ingestProperties.getMaxRecordsToWriteLocally())
-                .buildDirectWrite(ingestProperties.getFilePrefix() + ingestProperties.getBucketName());
+                .arrowRecordBatchFactory(builder -> builder
+                        .bufferAllocator(arrowBufferAllocator)
+                        .maxNoOfRecordsToWriteToArrowFileAtOnce(maxNoOfRecordsToWriteToArrowFileAtOnce)
+                        .workingBufferAllocatorBytes(workingArrowBufferAllocatorBytes)
+                        .minBatchBufferAllocatorBytes(minBatchArrowBufferAllocatorBytes)
+                        .maxBatchBufferAllocatorBytes(maxBatchArrowBufferAllocatorBytes)
+                        .maxNoOfBytesToWriteLocally(ingestProperties.getMaxRecordsToWriteLocally()))
```

Comment:
This is incorrect, the records to write locally and bytes to write locally are two separate things.